Gemfile.lock

```diff
@@ -363,12 +363,12 @@ GEM
       disposable (~> 0.0.5)
       representable (~> 2.1.0)
       uber (~> 0.0.11)
-    representable (2.1.5)
+    representable (2.1.8)
       multi_json
       nokogiri
       uber (~> 0.0.7)
     request_store (1.1.0)
-    roar (1.0.0)
+    roar (1.0.1)
```

Comment:
@NobodysNightmare nothing against updating the gems but is the update require for the PR?